cmd/cli/main.go

```diff
@@ -5,7 +5,8 @@ import (
 	"log"
 	"octavius/internal/cli/command"
 	"octavius/internal/cli/config"
-	"octavius/internal/cli/daemon"
+	job_daemon "octavius/internal/cli/daemon/job"
+	metadata_daemon "octavius/internal/cli/daemon/metadata"
```

Comment:
do not use camel_case for package name

>Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns.

https://blog.golang.org/package-names